### PR TITLE
Modifications to add events

### PR DESCRIPTION
**Pull request by MRScheid**
- Added event_times.csv which contains the reach event times for all subjects.
- Added code to add the appropriate events to each nwb file from this event_times.csv.
- Fixed some bugs in the filepath construction in the convert_dir function.

### CODE REVIEW

**Comment by MRScheid** [comment]
I tested the modifications I made in this branch on one nwb file and it looks like it processed ok.  Waiting to see the results of the conversion of two more h5 files.

File: brunton_lab_to_nwb/brunton_lab_to_nwb.py [hunk 1]
```diff
@@ -203,10 +220,11 @@ def convert_dir(in_dir, n_jobs=1, overwrite: bool = False):
...
 
     if overwrite:
-        in_files = [str(in_dir / f"{x}.h5") for x in all_data_files]
+        in_files = [str(in_dir + '\\' + f"{x}.h5") for x in all_data_files]
     else:
-        in_files = [str(in_dir / f"{x}.h5") for x in all_data_files if x not in nwb_files]
-    out_files = [str(in_dir / f"{Path(x).stem}.nwb") for x in in_files]
+        print(all_data_files, in_dir)
+        in_files = [str(in_dir + '\\' + f"{x}.h5") for x in all_data_files if x not in nwb_files]
+    out_files = [str(in_dir + '\\' + f"{Path(x).stem}.nwb") for x in in_files]
```
Comment (bendichter) [inline brunton_lab_to_nwb/brunton_lab_to_nwb.py:228]: This looks like a windows vs. linux issue. Use `os.path.join` or `pathlib.Path` objects so this works in any OS

**Review by bendichter — CHANGES_REQUESTED** [review]
Looks good. Can you fix the path stuff mention and also can you remove the .csv from the PR? Data should not be included an any git commits.

File: brunton_lab_to_nwb/brunton_lab_to_nwb.py (hunk 1 above)
Comment (MRScheid) [reply brunton_lab_to_nwb/brunton_lab_to_nwb.py:228]: ah, sorry about that--I saw the elec_loc_labels.csv in the repo and see it in being loaded directly from the top level in the package so I thought that I should follow suit and include this for the conversion of the entire dataset.  Whoever converts the rest of the files and inserts the event info will need this file and edit the path on line 198 to point to it.

Comment (bendichter) [reply brunton_lab_to_nwb/brunton_lab_to_nwb.py:228]: you should edit the call signature of `run_conversion` to include the events file

**Pull request merged**